ui_framework/doc_site/src/views/card/card_group.js

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+
+import {
+  KuiCardGroupContainer,
+  KuiCardGroup,
+  KuiCard,
+  KuiCardDescription,
+  KuiCardDescriptionTitle,
+  KuiCardDescriptionText,
+  KuiCardFooter
+} from '../../../../components';
+
+export default () => {
+  const cardStyle = {
+    width: '400px'
+  };
+
+  return (
+    <div>
+      <KuiCardGroupContainer>
+        <KuiCardGroup>
+          <KuiCard className="kuiCardGroup__card" style={cardStyle}>
```

Comment:
Also, `ui_framework/dist/ui_framework.css` is a compiled file, so it's the result of compiling `ui_framework/components/card/_card_group.scss`. It needs to get checked into PRs, but you can ignore it otherwise.